src/components/formatting/inputRow.tsx

```diff
@@ -1,25 +1,45 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import FileUpload from '../fileUpload/FileUpload';
+import DropDown from '../fileImport/fileImport';
 
 interface Props {
   onMeasureUpload: (files: any) => void;
   onPatientUpload: (files: any) => void;
   measureFileName: string | null;
   patientFileName: string | null;
+  onMeasureDropdownChange: (event: React.ChangeEvent<{ value: unknown }>) => void;
+  onPatientDropdownChange: (event: React.ChangeEvent<{ value: unknown }>) => void;
+  measureOptions: string[];
+  patientOptions: string[];
 }
+
 export default function InputRow(props: Props) {
   return (
     <React.Fragment>
       <Grid item xs={6}>
         <h2>Measure Bundle: </h2>
-        <h3>{props.measureFileName}</h3>
-        {props.measureFileName === null && <FileUpload onDrop={props.onMeasureUpload} />}
+        {props.patientFileName !== null && <h3>Current Measure Bundle:</h3>}
+        <h3 style={{ color: '#3f51b5' }}> {props.measureFileName}</h3>
```

Comment:
Can you do the style here using the makeStyles thing that we do for others? that way if the color changes, we only need to change it in one place rather than 2 (see below)